### ISSUE TITLE
[network-topology] Add cron to run `update_topology` command

### ISSUE THREAD

**Issue by nemesisdesign** [openwisp/ansible-openwisp2#20]
#19 introduced the openwisp-network-topology module developed by @rohithasrk but we forgot to add the cronjob to run `./manage.py update_topology` which is needed to automatically update the topologies using the FETCH strategy every `x` minutes.

When `openwisp2_network_topology` is set to `true` we should add a cron that runs the `update_topology` command, eg:

```
3 * * * * /root/.virtualenvs/ninux_graph/bin/python /var/www/ninux_graph/manage.py update_topology
```

In order to control the cron frequency, we could copy the syntax used by the role [finn.letsencrypt](https://github.com/thefinn93/ansible-letsencrypt/blob/master/tasks/main.yml#L102-L109) (which [we suggest in our README to handle the SSL cert](https://github.com/openwisp/ansible-openwisp2#automatic-ssl-certificate)).

**Comment by rohithasrk** [comment]
This shall be added once the PRs in **django-netjsongraph** and **openwisp-network-topology** are merged.

**Comment by nemesisdesign** [comment]
Closed by #21
